app/src/main/java/net/aiscope/gdd_app/ui/policy/PrivacyPolicyActivity.kt

```diff
@@ -0,0 +1,118 @@
+package net.aiscope.gdd_app.ui.policy
+
+import android.annotation.SuppressLint
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import android.webkit.JavascriptInterface
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import androidx.annotation.WorkerThread
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.isVisible
+import dagger.android.AndroidInjection
+import net.aiscope.gdd_app.R
+import net.aiscope.gdd_app.databinding.ActivityPrivacyPolicyBinding
+import net.aiscope.gdd_app.ui.main.MainActivity
+import javax.inject.Inject
+
+
+class PrivacyPolicyActivity : AppCompatActivity(), PrivacyPolicyView {
+
+    @Inject
+    lateinit var presenter: PrivacyPolicyPresenter
+
+    private lateinit var binding: ActivityPrivacyPolicyBinding
+
+    @SuppressLint("SetJavaScriptEnabled")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        AndroidInjection.inject(this)
+        super.onCreate(savedInstanceState)
+        binding = ActivityPrivacyPolicyBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        setSupportActionBar(binding.toolbar)
+
+        setUpListeners()
+        setUpWebView()
+
+        presenter.start()
+    }
+
+    private fun setUpListeners() = with(binding) {
+        error.setOnClickListener {
+            presenter.onRetryClicked()
+        }
+        agreeButton.setOnClickListener {
+            presenter.onPolicyAgreed()
+        }
+        cancelButton.setOnClickListener {
+            finish()
+        }
+    }
+
+    override fun goToMain() {
+        finish()
+        startActivity(Intent(this, MainActivity::class.java))
+    }
+
+    private fun setUpWebView() {
+        WebView.setWebContentsDebuggingEnabled(true)
+        with(binding.webView) {
+            settings.javaScriptEnabled = true
```

Comment:
No way to set this through the XML, right? (I googled for it but couldn't find anything)

Reply:
Nope, unfortunately :(